redbot/cogs/mod/kickban.py

```diff
@@ -210,15 +210,18 @@ class KickBanMixin(MixinMeta):
         self,
         ctx: commands.Context,
         user: discord.Member,
-        days: Optional[int] = 0,
+        days: Optional[int] = None,
         *,
         reason: str = None,
     ):
         """Ban a user from this server and optionally delete days of messages.
 
         If days is not a number, it's treated as the first word of the reason.
-        Minimum 0 days, maximum 7. Defaults to 0."""
-
+        Minimum 0 days, maximum 7. If defaultdays is set and days not used 
+        in command, the defaultdays will be used."""
```

Comment:
>If defaultdays is set and days not used in command, the defaultdays will be used.

This could use some rewording. Maybe something along the lines of:
```
If not specified, defaultdays will be used instead.
```